config/environments/development.rb

```diff
@@ -85,6 +85,7 @@
 
   # allow users to access this application via the configured application domain.
   config.hosts << default_url_options_from_base_url[:host]
+  config.hosts << /.*\.gitpod\.io/
```

Comment:
Is there any way of detecting whether we are running on a gitpod?

Reply:
Yes I believe there are environment variables that get set.  I'll add a check for them.